models/trigger.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Joi = require('joi');
+const Job = require('../config/job');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .example(12345),
+
+    pipelineId: Joi
+        .number().integer().positive()
+        .example(56789),
+
+    jobName: Job.jobname
+        .example('component'),
+
+    trigger: Joi
```

Comment:
Can we user the EXTERNAL_TRIGGER we just added to the regex? Just to be consistent